api/api.yaml

```diff
@@ -2615,6 +2615,7 @@ components:
         - createTime
         - id
         - name
+        - entitlements
```

Comment:
entitlements shouldn't be in in required since initially when this PR is merged, our projects won't be returning entitlements yet